common/src/main/java/io/quarkus/ts/openshift/common/deploy/UsingQuarkusPluginDeploymentStrategy.java

```diff
@@ -0,0 +1,98 @@
+package io.quarkus.ts.openshift.common.deploy;
+
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.Command;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * If the test class is using the {@code UsingQuarkusPluginDeploymentStrategy} strategy, the test framework will use the
+ * Quarkus OpenShift extension to deploy the Quarkus Application by using the property `quarkus.kubernetes.deploy`.
+ *
+ */
+public class UsingQuarkusPluginDeploymentStrategy implements DeploymentStrategy {
+
+    private static final String MVN_COMMAND = "mvn";
+    private static final String PACKAGE_GOAL = "package";
+    private static final String QUARKUS_PLUGIN_DEPLOY = "-Dquarkus.kubernetes.deploy=true";
+    private static final String SKIP_TESTS = "-DskipTests=true -DskipITs=true";
```

Comment:
This is passed as 1 argument, even though you clearly want to pass 2. If you really need to pass `-DskipITs`, it needs to be a separate argument.

Reply:
Having the embedded space was correctly interpreted by processbuilder. However, I've separated it into two fields "SKIP_TESTS" and "SKIP_ITS" to avoid misunderstandings ...